src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -64,7 +64,7 @@ public sealed class Features
 
         internal bool PowerFxV1CompatibilityRules { get; set; }
 
-        internal static Features None => new Features();
+        public static Features None => new Features();
```

Comment:
Does it need to be public? We had made it internal exactly to force people to use PowerFxV1 - and only let Power Apps take its time to catch up.